moreComfortable.js

```diff
@@ -0,0 +1,114 @@
+//two players will take turns to play
+//when the player clicks submit, game rolls 2 dice and show the values
+//player then can pick the order of dice they want ie: 36 or 63
+//after both players have rolled and chosen the dice order, the player with the highest combined number wins.
+//have to store the values of the chosen combination
```

Comment:
Good job using pseudocode to plan what need to happen before writing code. Keep up this habit!